app/controllers/api/v0/cromerr_submit_transactions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V0::CromerrSubmitTransactionsController < ApiController
```

Comment:
This controller name feels too long to me.

How about `Api::V0::SubmissionsController` instead?